styles/about.css

```diff
@@ -0,0 +1,335 @@
+    background-color: #000000;
+    border-width: 0;
+    transition: border-width 100ms 340ms ease, 
+            top 100ms 100ms cubic-bezier(0.23, 1, 0.32, 1),
+            height 100ms 100ms cubic-bezier(0.23, 1, 0.32, 1), 
+            background-color 100ms  100ms ease,
+            transform 100ms 100ms cubic-bezier(0.23, 100, 0.32, 1);
+}
+.menu-icon:checked + label:after{
+    width: 30px;
+    margin-top: 0;
+    right: 0;
+    transform: rotate(-45deg);
+    transition: width 100ms ease,
+            right 100ms ease,  
+            margin-top 100ms 100ms ease, 
+            transform 100ms 100ms cubic-bezier(0.23, 100, 0.32, 1);
+}
+
+.nav{
+    position: fixed;
+    top: 33px;
+    right: 50px;
+    display: block;
+    width: 80px;
+    height: 80px;
+    padding: 0;
+    margin: 0;
+    z-index: 9;
+    overflow: hidden;
+    box-shadow: 0 8px 30px 0 rgb(255, 255, 255);
+    background-color: #1500ff;
+    
+    animation: border-transform 7s linear infinite;
+    transition: 
+    top 350ms 1100ms cubic-bezier(0.23, 1, 0.32, 1),  
+    right 350ms 1100ms cubic-bezier(0.23, 1, 0.32, 1),
+    transform 250ms 1100ms ease,
+    width 650ms 400ms cubic-bezier(0.23, 1, 0.32, 1),
+    height 650ms 400ms cubic-bezier(0.23, 1, 0.32, 1);
+}
+
+
+@keyframes border-transform{
+    0%,100% { border-radius: 63% 37% 54% 46% / 55% 48% 52% 45%; }
+    
+    14% { border-radius: 40% 60% 54% 46% / 49% 60% 40% 51%; } 
+    28% { border-radius: 54% 46% 38% 62% / 49% 70% 30% 51%; } 
+    42% { border-radius: 61% 39% 55% 45% / 61% 38% 62% 39%; } 
+    56% { border-radius: 61% 39% 67% 33% / 70% 50% 50% 30%; } 
+    70% { border-radius: 50% 50% 34% 66% / 56% 68% 32% 44%; } 
+    84% { border-radius: 46% 54% 50% 50% / 35% 61% 39% 65%; } 
+}
+
+.menu-icon:checked ~ .nav {
+    animation-play-state: paused;
+    top: 50%;
+    right: 50%;
+    transform: translate(50%, -50%);
+    width: 200%;
+    height: 200%;
+    transition: top 350ms 700ms cubic-bezier(0.23, 1, 0.32, 1), 
+    right 350ms 700ms cubic-bezier(0.23, 1, 0.32, 1),
+    transform 250ms 700ms ease,
+    width 750ms 1000ms cubic-bezier(0.23, 1, 0.32, 1),
+    height 750ms 1000ms cubic-bezier(0.23, 1, 0.32, 1);
+}
+
+.nav ul{
+    position: absolute;
+    top: 50%;
+    left: 0;
+    display: block;
+    width: 100%;
+    padding: 0;
+    margin: 0;
+    z-index: 6;
+    text-align: center;
+    transform: translateY(-50%);
+    list-style: none;
+}
+.nav ul li{
+    position: relative;
+    display: block;
+    width: 100%;
+    padding: 0;
+    margin: 10px 0;
+    text-align: center;
+    list-style: none;
+    pointer-events: none;
+    opacity: 0;
+    visibility: hidden;
+    transform: translateY(30px);
+    transition: all 250ms linear;
+}
+.nav ul li:nth-child(1){
+    transition-delay: 200ms;
+}
+.nav ul li:nth-child(2){
+    transition-delay: 150ms;
+}
+.nav ul li:nth-child(3){
+    transition-delay: 100ms;
+}
+.nav ul li:nth-child(4){
+    transition-delay: 50ms;
+}
+.nav ul li a{
```

Comment:
This selector is getting a bit complex. You might consider using a more specific class name to simplify here.